src/component/scss/components.scss

```diff
@@ -45,10 +45,10 @@
 
 .undobox {
     display : block;
-    position : fixed;
+    position : fixed!important;
```

Comment:
1. I don't like this solution with `!important` usage. probably changing scss file ordering make a trick: if you load _feedback.scss and then component.scss
2. when you creating new version of the fe package you have to compile css file, that should appear in the dist directory (your case is dist/component/css/component.css)

Reply:
done